Configuration/Objects.yaml

```diff
@@ -15,6 +15,16 @@ Sitegeist\LostInTranslation\Infrastructure\DeepL\DeepLTranslationService:
           1:
             value: Sitegeist_LostInTranslation_TranslationCache
 
+Sitegeist\LostInTranslation\Infrastructure\DeepL\DeepLAuthenticationKeyFactory:
+  properties:
+    apiKeyCache:
+      object:
+        factoryObjectName: Neos\Flow\Cache\CacheManager
+        factoryMethodName: getCache
+        arguments:
+          1:
+            value: Sitegeist_LostInTranslation_ApiKeyCache
+
 Sitegeist\LostInTranslation\Controller\LostInTranslationModuleController:
   properties:
```

Comment:
Should this be removed ... looks like Sitegeist_LostInTranslation_ApiKeyCache is now configured twice

Reply:
No, this section configures `$this->apiKeyCache` for the `DeepLAuthenticationKeyFactory`. The other configurations are also still relevant, as we access the api key cache there as well.

Reply:
That smells like an incomplete refactoring. Ideally only the  DeepLAuthenticationKeyFactory would have to know anything about that cache.

Reply:
I now moved everything cache related to a new singleton class `DeepLCustomAuthenticationKeyService`. This is the new centerpoint for retrieving and saving the custom key.

Also, I already adjusted parts of the code in this PR to PHP 8.1.